### ISSUE TITLE
Request:  Support other Email/"Accounts"

### ISSUE THREAD

**Issue by laurielamberth** [MetaWatchOpenProjects/MWM-for-Android-Gen1#10]
I receive email from three separate email accounts on my Droid X2:  from gmail, my own business, and my primary client's business.  The two non-Gmail accounts have been set up on my Droid using the "Accounts" feature -- therefore, since Android supports them, I believe the Metawatch can, too.

Please add support for notifications from other email accounts that are established using the standard Android "Accounts" feature (instead of through some other application...  this is a native Android capability)

Thanks

**Comment by ceeeKay** [comment]
Yes, please! I have my work email (Exchange) that I check using the normal android Email program, and my Gmail account. I'd really like to know my email count for either/both.

**Comment by serountree** [comment]
depending on exchange versions, you have a couple of options that I found out about. First, I believe that MWM is designed to use the K9 email app found in the market. I had downloaded it when I found out, and was able to use it to receive notifications from my home email (POP3) and Gmail accounts, and it can handle some Exchange servers.

The downside is that K9 (the last I checked) cannot handle interfacing with MS Exchange 2010, as Micro$oft apparently dropped support for an older standard that K9 used, and K9 doesn't yet work with the new standard. Benjymous in the Metawatch forums did help me with a work around, by going into the "Other Apps" section of MWM and telling it to watch for notifications there. It won't show how many messages you have on the watch, but it does allow the watch to vibrate when a message comes in. It isn't perfect, but at least until K9 gets updated it's let me know when a message comes in and to check my phone. 

Hope this helps.

**Comment by ceeeKay** [comment]
Supporting only Gmail and one non-native app (K9) is what I'd consider the issue here. Support for a popular third-party app is great, but most people are likely using the default mail app for at least one account.

**Comment by sdemills** [comment]
I would like to lend my support here. I too have Exchange email for work and it is not supported by K9 and I can't get alerts on my MetaWatch.

I have done quite a lot of research into this and it would appear that the main problem is that the email notification that is sent out has no tag attached to it that MWM might use to grab it and create an alert.

An idea I had been exploring was to write a Tasker script that would detect the email and send a message alert to the watch through MWM. I did have a partial success with that, but again the lack of a usable tag was preventing an elegant solution. My "solution" was based on analysing the content of a Notification that has no tag.

Both MWM and Tasker have features to enable the App to detect messages/notifications from other existing Apps, and they both do this from a user point of view by presenting a list of App icons from which you can choose, in both cases the email App in question is there on the list, yet still we seem unable to respond to its notifications.

Until recently the Tasker-based solution was working but that is no longer the case (for reasons as yet unknown).

Even when that solution does work, it seems I can't get things like the count of how many emails are waiting to be read, but at least an alert that one has arrived was useful whilst it was working.
